src/components/ScoreCardEmoji/ScoreCardEmoji.style.tsx

```diff
@@ -5,11 +5,19 @@ export const ScoreCardEmojiImageLeft = styled.img`
   top: 50%;
   left: -3.5rem;
   translate: 50% -50%;
+  @media (min-width: 1040px) {
+    top:0;
+    left:3.6rem;
```

Comment:
Please add a space after "top:" and "left:" and set the left property to a percentage value.